models/articles.model.js

```diff
@@ -2,7 +2,12 @@ const format = require("pg-format");
 const db = require("../db/connection");
 
 exports.selectAllArticles = () => {
-  return db.query(`SELECT * FROM articles`);
+  return db
+    .query(
+      `SELECT * FROM articles
+      ORDER BY created_at DESC;`
+    )
```

Comment:
you are going to need to join the tables here as you need the comment count too which is on a different table - tip: you will need a left join for this 👀

Reply:
Sorry for any confusion, but this is for task 5 of the sprint, I had done two more pull requests than needed so it now says ticket 7 but for task 5. Wondering if there is a way I can change it?